docs/content/docs/custom-resource/job-management.md

```diff
@@ -288,12 +288,33 @@ Rollback is currently only supported for `FlinkDeployments`.
 
 ## Manual Recovery
 
-There are cases when manual intervention is required from the user to recover a Flink application deployment.
+There are cases when manual intervention is required from the user to recover a Flink application deployment or to restore to a user specified state.
 
 In most of these situations the main reason for this is that the deployment got into a state where the operator cannot determine the health of the application or the latest checkpoint information to be used for recovery.
 While these cases are not common, we need to be prepared to handle them.
 
-Fortunately almost any issue can be recovered by the user manually by using the following steps:
+Users have two options to restore a job from a target savepoint / checkpoint
+
+### Redeploy using the savepointRedeployNonce
+
+It is possible to redeploy a `FlinkDeployment` or `FlinkSessionJob` resource from a target savepoint by using the combination of `savepointRedeployNonce` and `initialSavepointPath` in the job spec:
+
+```yaml
+ job:
+   initialSavepointPath: file://redeploy-target-savepoint
+   savepointRedeployNonce: null -> 1
```

Comment:
I think this is irritating. How about adding a comment? 
```suggestion
   # If not set previously, set to 1, otherwise increment, e.g. 2
   savepointRedeployNonce: 1
```

Reply:
We use the `a -> b` notation in logs where we display spec changes. I think it's concise and intuitive. Definitely would not call it "irritating" :)

Reply:
If I were a new user, I would copy this as-is. I'm suspecting it will fail. My suggestion would work fine.

Reply:
alright, I will update it to avoid unexpected copy-paste behaviour

Reply:
Please only update if you agree, not intending to block the PR on this.